node/README.md

```diff
@@ -0,0 +1,52 @@
+# WebNN-native Binding for Node.js
+
+*WebNN-native Binding for Node.js is a Node.js [C++ addon](https://nodejs.org/api/addons.html). The implementation is based on WebNN-native C API and exposes [WebNN](https://webmachinelearning.github.io/webnn/) JavaScript API.
```

Comment:
I suppose it is based on the WebNN-native C++ API now. Please correct me if I am wrong.